### PR TITLE
Fix issue with IoT Hub key

### PR DESCRIPTION
**Pull request by vicmgs**

### CODE REVIEW

File: assets/js/components/channels/forms/AzureHubForm.jsx [hunk 1]
```diff
@@ -58,14 +58,23 @@ class AzureHubForm extends Component {
       } else {
         const credentials = {};
         e.target.value.split(";").forEach((x) => {
-          const [key, value] = x.split("=");
+          const tuple = x.split("=");
```
Comment (ambuznego) [inline assets/js/components/channels/forms/AzureHubForm.jsx:61]: I was going to fix by just doing `.split(/=(.*)/s);` instead which will only look at the first occurrence of `=`. Might make it a little more straightforward to read this for posterity. Thoughts?

Comment (ambuznego) [reply assets/js/components/channels/forms/AzureHubForm.jsx:61]: tested:

```
➜  console git:(dev) ✗ node
Welcome to Node.js v14.15.5.
Type ".help" for more information.
> let credentials = {};
undefined
> let someString = "HostName=myname;SharedAccessKeyName=word;SharedAccessKey=wow=2482";
undefined
> let someFunction = (input) => { input.split(";").forEach((x) => {
...           const [key, value] = x.split(/=(.*)/s);
...           if (
.....             ["HostName", "SharedAccessKeyName", "SharedAccessKey"].indexOf(
.......               key
.......             ) > -1 &&
.....             valueOf
.....           ) {
.....             credentials[key] = value;
.....           }
...         });};
undefined
> someFunction(someString);
undefined
> credentials
{
  HostName: 'myname',
  SharedAccessKeyName: 'word',
  SharedAccessKey: 'wow=2482'
}
```

Comment (vicmgs) [reply assets/js/components/channels/forms/AzureHubForm.jsx:61]: Great idea, il change it out

**Review by ambuznego — APPROVED** [review]
nugget-approved.

**Pull request merged**